languages/elixir/exercises/concept/booleans/.docs/instructions.md

```diff
@@ -0,0 +1,41 @@
+In this exercise, you need to translate some rules from the classic game Pac-man into elixir functions.
+
+You have four rules to translate, all related to the game states.
+
+> Don't worry about how the paramaters are derived, just focus on combining the parameters to return the intended result.
+
+### 1. Define if pac-man can eat a ghost
```

Comment:
```suggestion
### 1. Define if pac-man eats a ghost
```
This was a little confusing to me because technically if you want to know if pac man _can_ (or is able to) eat a ghost, you only need to know if he ate a pill.

Reply:
I don't have strong feelings about this wording change, I can change it if you like.